### PR TITLE
Snap zone

### PR DESCRIPTION
**Pull request by Abb-FilGal**
Snap zones are a new prefab that can check for specific items. Right now the boolean of if it is the right item will be shown by the color of the snap zone. Size and shape should be adjustable and new ones can be created within minutes for specific purposes. 
The todolist/score is also easy to track with a simple if-statement.

### CODE REVIEW

**Review by abbsimmei — APPROVED** [review]
I want these changes! So I am approving!

File: VR_RV/Assets/Global Scripts/SnapZoneCheckItemName.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.Interaction.Toolkit;
+
+public class SnapZoneCheckItemName : MonoBehaviour
+{
+    public XRBaseInteractable correct_item;
+
+    public void OnSelectEnteredSnap(SelectEnterEventArgs args)
```
Comment (tramvaj99) [inline VR_RV/Assets/Global Scripts/SnapZoneCheckItemName.cs:10]:
```suggestion
    public void On_Select_Entered_Snap(SelectEnterEventArgs args)
```

File: VR_RV/Assets/Global Scripts/SnapZoneCheckItemName.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            GetComponent<MeshRenderer>().material.color = Color.green;
+            // TODO: update the score here;
+        }
+        else
+        {
+            GetComponent<MeshRenderer>().material.color = Color.red;
+        }
+    }
+
+    public void OnSelectExitedSnap(SelectExitEventArgs args)
```
Comment (tramvaj99) [inline VR_RV/Assets/Global Scripts/SnapZoneCheckItemName.cs:23]:
```suggestion
    public void On_Select_Exited_Snap(SelectExitEventArgs args)
```

**Review by tramvaj99 — CHANGES_REQUESTED** [review]
Ser bra ut, kom bara ihåg title case för funktioner :)

**Pull request merged**